MapboxNavigation/RouteVoiceController.swift

```diff
@@ -106,7 +108,7 @@ open class RouteVoiceController: NSObject, AVSpeechSynthesizerDelegate, AVAudioP
     func pauseSpeechAndPlayReroutingDing(notification: NSNotification) {
         speechSynth.stopSpeaking(at: .word)
         
-        guard playRerouteSound && !NavigationSettings.shared.muted else {
+        guard playRerouteSound && volume > 0 else {
```

Comment:
This is a nit-pick, and isn't even really in scope, but meh.


Generally I am a fan of the following convention.

* Use a _comma_ if the guard merely checks a list of preconditions. `This, that, and the other.`
* Use _boolean operators_ if the conditional uses non-vanilla boolean logic. `This AND that, OR this AND the other.`